### PR TITLE
Updated 'created repository' style

### PR DESCRIPTION
**Pull request by delvedor**
Added the rules for display the `user created repository user/repo` news as the others.
*I've commented some rules because they were overwriting the correct style.*

I've made all the work with the Chrome dev tools and for me everything is ok, but it's better if you check it as well. 😄

### CODE REVIEW

File: extension/content.css [hunk 1]
```diff
@@ -242,18 +242,46 @@ forked a repo
 remove padding from body of simple news alerts in Dashboard since some will be hidden
 we will add it back on for the simple news alerts we decide to show
 */
+/* - Disabled because these rules were overwriting other padding rules, breaking the layout. -
```
Comment (paulmolluzzo) [inline extension/content.css]: Disabling this introduces a bug. We are hiding notifications when creating a branch, but we also need to remove the padding so there's no extra spacing. See screenshot:

screen shot 2016-05-15 at 10 01 34 pm

Also, if you find a solution that doesn't include this CSS (or other CSS), you should delete it not comment it out.

**Comment by paulmolluzzo** [comment]
@delvedor Thanks for the PR! Could you provide a screenshot? No one I'm following has made a repo recently...

**Comment by delvedor** [comment]
Ok, here is a bit of screenshots:

**How is at the moment:**
screen shot 2016-05-16 at 10 14 04 am
___
**How is with my changes and with padding setted to 0:**
screen shot 2016-05-16 at 10 23 36 am
___
**How is with the actual padding of 1rem and my changes:**
screen shot 2016-05-16 at 10 23 54 am

In my commit I saw that I wrote `.created` instead of  `.create`, with next commit I will update it.

**Comment by delvedor** [comment]
I just saw that the `user made user/repo public` has the correct refined-style, and as you can see, is "shifted down" compared to the *star news*.
screen shot 2016-05-16 at 10 07 16 pm

**Comment by sindresorhus** [comment]
The `repo created` item has too much top padding.

**Comment by delvedor** [comment]
I agree, I don't understand if I can remove [these](https://github.com/delvedor/refined-github/blob/created-repo-patch/extension/content.css#L241-260) lines, because that code is hiding the `create repo` news.

**Comment by paulmolluzzo** [comment]
@delvedor You can't just remove those lines, not without replacing what they accomplish. :( Also, the CSS seems to be too general, and I get half styling on `[user] created [tag]` notifications.

screen shot 2016-05-22 at 10 09 38 pm

If you want to look through the relevant PR and issue that got those in there, they are https://github.com/sindresorhus/refined-github/pull/189 and https://github.com/sindresorhus/refined-github/issues/184

[This comment](https://github.com/sindresorhus/refined-github/pull/189#issuecomment-214289368) is what forced us to add in the specific CSS you're referring to.

The underlying idea is that we can't hide a `.simple` news item (or any HTML) with CSS based on the contents. So what we did was we moved the style for these elements to the content of the item (that's what [these lines](https://github.com/delvedor/refined-github/blob/created-repo-patch/extension/content.css#L241-260) do, and then we [hide the content if the news item is for creating a new branch](https://github.com/delvedor/refined-github/blob/created-repo-patch/extension/content.css#L286-291). The sum effect is that you only don't see "[user] created [branchname]" notifications and you also don't get extra empty space because of leftover `padding` when we don't show a news event.

I think you just need to leave the existing code and then override that padding accordingly with more specificity on the news elements you want to change (for new repo).

**Comment by alexbooker** [comment]
I cannot wait for this to land :smile:

**Comment by delvedor** [comment]
Hi, at the moment I've not time to to this due to university exams.
If you want you can assign this task to someone else or wait a little bit, sorry!

**Comment by paulmolluzzo** [comment]
I don't mind waiting so you can finish what you started.

**Comment by sindresorhus** [comment]
@delvedor Still interested?

**Comment by delvedor** [comment]
@sindresorhus sure, sorry but I'm very busy, I will work on this as soon as possible. :)

**Comment by delvedor** [comment]
@sindresorhus @paulmolluzzo 
Update: I'm so sorry, but at the moment I've not enough time to work on this.
If anyone want to work on this, go ahead.
Otherwise when I have a bit 'of time I'll finish this it.
Sorry again.

**Comment by devoxel** [comment]
Don't think there's a super rush, I'll take a look at your code if I get time 👍

**Comment by delvedor** [comment]
Is not a matter of super rush, but respect to you guys :)

**Comment by hkdobrev** [comment]
Just a reminder if anyone in this thread wants to pick this up, #213 is still open.

**Pull request closed without merging**